.github/workflows/rav1e.yml

```diff
@@ -280,9 +278,14 @@ jobs:
         $NASM_VERSION="nasm-2.14.02"
         curl -LO "https://people.xiph.org/~tdaede/$NASM_VERSION-win64.zip"
         7z e -y "$NASM_VERSION-win64.zip" -o"C:\nasm"
-    - name: Set environment variables
+        echo "::add-path::C:\nasm"
+    - name: Set MSVC x86_64 linker path
+      shell: pwsh
       run: |
-        echo "::add-path::$Env:VS_PATH\$Env:COMPILER_SUBPATH;C:\nasm"
+        $LinkGlob = "VC\Tools\MSVC\**\bin\Hostx64\x64\link.exe"
```

Comment:
Also, what's the reasoning behind separating out the glob string as a variable? Seems like an unnecessary line, not that it affects time very much

Reply:
This is for readability. The query command was getting quite long with the glob embedded.

Reply:
I see, seemed a bit weird to have a variable just for that since a lot of powershell commands can easily go over 100 characters anyway